apps/meteor/ee/app/settings/server/settings.ts

```diff
@@ -16,7 +16,7 @@ export function changeSettingValue(record: ISetting): SettingValue {
 	}
 
 	if (!record.modules?.length) {
-		return record.invalidValue;
+		return record.value;
```

Comment:
can you please explain this change? `invalidValue` should be the correct value to use here.

Reply:
@sampaiodiego It was not the behavior before @ggazzo refactor of the settings API. Also, it implies all enterprise settings *MUST HAVE* a list of modules they are contained. Is it right? If so, which module fits here?

Reply:
oh, I see, this check is just in case an enterprise setting doesn't have a list of modules.. since there is an `isEnterprise()` validation before, I'm ok with this change then.. 

initially it wouldn't make sense have an enterprise license without modules defined, because there is no enterprise license without a module, but I can see it may be a thing.